src/components/dropdown.component.js

```diff
@@ -4,15 +4,51 @@ export default class DropdownComponent {
     constructor(query) {
         this.query = query;
 
+        this.dropdownContainer = $(`${query} .dropdown-container`);
         this.btn = $(`${query} .dropdown-btn`);
-        this.items = $(`${query} .dropdown-item`);
         this.list = $(`${query} .dropdown-list`);
+        this.items = $(`${query} .dropdown-item`);
         this.selectedItem = $(`${query} .selected`);
 
+        this.enabled = true;
+
         this.addDropdownClickEventListner();
         this.addItemClickEventListner();
     }
 
+    getSelectedItem() {
+        return this.selectedItem.text();
+    }
+
+    checkQuery() {
+        if (
```

Comment:
Why do you need an if/else to return boolean values?

Reply:
I've removed this checkQuery.